nimbus/src/lib.rs

```diff
@@ -89,6 +93,17 @@ impl NimbusClient {
     pub fn update_experiments(&self) -> Result<()> {
         unimplemented!()
     }
+
+    pub(crate) fn nimbus_id(&self) -> Result<Uuid> {
```

Comment:
Until we have the Glean Rust API available, can we make this a `pub` function so that the android-component can access the ID to record in telemetry? I'm happy to have a follow-up issue when we migrate to recording telemetry all in the Rust component to add back the crate-only visibility.